packages/lesswrong/components/form-components/ImageUpload.tsx

```diff
@@ -115,7 +121,11 @@ const formPreviewSizeByImageType = {
   spotlightImageId: {
     width: 345,
     height: 234
-  }
+  },
+  spotlightDarkImageId: {
```

Comment:
Do you have an expectation that this class will ever be different from the spotlightImageId class? Seems probably better to just have one class unless there's a specific reason.

Reply:
This isn't a class. It's the width and height to pass into CloudinaryImage2 as props, keyed by the image type. I could define a variable,

```ts
const spotlightImageDimensions = {
    width: 345,
    height: 234
}
```

I could use that here:

```ts
  spotlightImageId: spotlightImageDimensions,
  spotlightDarkImageId: spotlightImageDimensions,
```

But that seems to increase the reading complexity to get a small amount DRY, and I don't think it's worth it.

Or below I could do   

```ts
const formPreviewSize = formPreviewSizeByImageType[name === 'spotlightDarkImageId' ? 'spotlightImageId' : name]
```

But I once again don't think it's worth it, for similar reasons.